InterruptedException in PerformanceThread.run()

Happens from time to time when using external clients (don't know if it will fail on regular tests as well). It's after the testsuite completes, so performance results and probes are written. Maybe it's just not shutdown properly.
```
FATAL 13:58:43 Exception in PerformanceThread.run()
java.lang.RuntimeException: java.lang.InterruptedException
	at com.hazelcast.simulator.coordinator.remoting.AgentsClient.getAllFutures(AgentsClient.java:394)
	at com.hazelcast.simulator.coordinator.remoting.AgentsClient.executeOnAllWorkersDetailed(AgentsClient.java:333)
	at com.hazelcast.simulator.coordinator.PerformanceMonitor$PerformanceThread.checkPerformance(PerformanceMonitor.java:110)
	at com.hazelcast.simulator.coordinator.PerformanceMonitor$PerformanceThread.run(PerformanceMonitor.java:99)
Caused by: java.lang.InterruptedException
	at java.util.concurrent.FutureTask.awaitDone(FutureTask.java:404)
	at java.util.concurrent.FutureTask.get(FutureTask.java:204)
	at com.hazelcast.simulator.coordinator.remoting.AgentsClient.getAllFutures(AgentsClient.java:381)
	... 3 more
```

This might be related:
```
Exception in thread "Thread-2" java.lang.RuntimeException: java.lang.InterruptedException
	at com.hazelcast.simulator.coordinator.remoting.AgentsClient.getAllFutures(AgentsClient.java:394)
	at com.hazelcast.simulator.coordinator.remoting.AgentsClient.executeOnAllWorkers(AgentsClient.java:308)
	at com.hazelcast.simulator.coordinator.TestCaseRunner$StopThread.run(TestCaseRunner.java:264)
Caused by: java.lang.InterruptedException
	at java.util.concurrent.FutureTask.awaitDone(FutureTask.java:404)
	at java.util.concurrent.FutureTask.get(FutureTask.java:204)
	at com.hazelcast.simulator.coordinator.remoting.AgentsClient.getAllFutures(AgentsClient.java:381)
	... 2 more
```

--- a/simulator/src/main/java/com/hazelcast/simulator/coordinator/remoting/AgentsClient.java
+++ b/simulator/src/main/java/com/hazelcast/simulator/coordinator/remoting/AgentsClient.java
@@ -11,6 +11,7 @@ import com.hazelcast.simulator.test.Failure;
 import com.hazelcast.simulator.test.TestPhase;
 import com.hazelcast.simulator.test.TestSuite;
 import com.hazelcast.simulator.utils.CommandLineExitException;
+import com.hazelcast.simulator.utils.EmptyStatement;
 import com.hazelcast.simulator.worker.commands.Command;
 import com.hazelcast.simulator.worker.commands.IsPhaseCompletedCommand;
 import org.apache.log4j.Logger;
@@ -391,7 +392,9 @@ public class AgentsClient {
                 }
                 throw new RuntimeException(e);
             } catch (InterruptedException e) {
-                throw new RuntimeException(e);
+                LOGGER.info("Got interrupted while waiting for results from Agents!");
+                EmptyStatement.ignore(e);
+                break;
             }
         }
         return resultList;

